LICENSE-binary

```diff
@@ -285,7 +285,8 @@ com.zaxxer:HikariCP-java7:2.4.13
 commons-cli:commons-cli:1.5.0
 commons-codec:commons-codec:1.15
 commons-collections:commons-collections:3.2.2
-commons-configuration:commons-configuration:1.10
+commons-configuration:commons-configuration2:2.9.0
+commons-beanutils:commons-beanutils:1.9.4
```

Comment:
(minor) We should keep it alphabetical order (move this in front of `commons-cli:commons-cli:1.5.0`)